### DISCUSSION TITLE
Query from Java on smartphone to smartwatch

### DISCUSSION THREAD

**Discussion by mazezone** [zepp-health/zeppos-forum#302] [Q&A]
Is it possible to implement running a specific application in ZeppOS on a watch by ID from Android environment? I would like from the Android side in Java to be able to access the smartwatch via the Zepp app.

**Comment by MyoungXUE** [comment]
Sorry, currently Zepp App does not have such an interface, but you can use the server side. This is our architecture diagram to communicate with the server side.

https://docs.zepp.com/docs/guides/architecture/arc/
